### PR TITLE
run trigger-workflow-dispatch action using composite instead of docker

### PR DESCRIPTION
**Pull request by dorsradek**
run trigger-workflow-dispatch action using composite instead of docker

### CODE REVIEW

File: trigger-workflow-dispatch/action.yaml [hunk 1]
```diff
@@ -20,11 +20,13 @@ inputs:
...
 runs:
-  using: 'docker'
-  image: 'Dockerfile'
-  args:
-    - ${{ inputs.GITHUB_DISPATCH_WORKFLOW_TOKEN }}
-    - ${{ inputs.WORKFLOW_DISPATCH_FILE_NAME }}
-    - ${{ inputs.ENV }}
-    - ${{ inputs.WORKFLOW_FILE_REF }}
-    - ${{ inputs.ALLOW_CHAINED_DEPLOYMENTS }}
+  using: 'composite'
```
Comment (braedongough) [inline trigger-workflow-dispatch/action.yaml:23]: What is composite?

**Review by alexhumphreys — APPROVED** [review]

File: trigger-workflow-dispatch/action.yaml (hunk 1 above)
Comment (bittelc) [reply trigger-workflow-dispatch/action.yaml:23]: ^

Comment (dorsradek) [reply trigger-workflow-dispatch/action.yaml:23]: there are 3 ways to run gh actions:
- docker
- javascript
- composite - allows to run bash script directly in the runner

Comment (braedongough) [reply trigger-workflow-dispatch/action.yaml:23]: TIL

**Review by braedongough — APPROVED** [review]

**Pull request merged**